src/service/fixed-layer.js

```diff
@@ -307,7 +313,7 @@ export class FixedLayer {
           // `height` is constrained to at most 300px. This is to avoid
           // transfering of more substantial sections for now. Likely to be
           // relaxed in the future.
-          const isTransferrable = (
+          const isTransferrable = fe.forceTransfer || (
```

Comment:
Since you added `forceTransfer` already in `isFixed`, you should defer to `isFixed` here. E.g. it should likely look like:

```
const isTransferrable = isFixed && (
     fe.forceTransfer || (opacity > 0 && ...)
  );
```